### ISSUE TITLE
Comments under sources don't appear anywhere visible to other users?

### ISSUE THREAD

**Issue by commonplace** [aaroneiche/do-want#113]
This might not be an actual issue, just me not understanding the purpose of something. But when you add sources to an item, there's a Comments field. For years, I've added comments for most things I add, like "Amazon Prime eligible" or "Free in-store pickup available" etc.

I never noticed or realized that no one can actually see these things I write. They don't show up anywhere other than to myself when I edit the source for the item in question.

Is that... right? What's the purpose of the Comments field for sources? Are they just notes to myself?

**Comment by aaroneiche** [comment]
You're right, and I kind of noticed this the other night when I was looking at my own stuff. Those comments are (or were initially) intended for other people. I don't know why they're not showing up. Fortunately, it should be an easy fix.

**Comment by aaroneiche** [comment]
Okay, I just tested it on my "production" install, and it's working there, but I believe that it's not working on my dev system. So I'll have to poke around and figure out what's going on.
